src/index.js

```diff
@@ -1,11 +1,45 @@
-import _ from 'lodash';
+// src/index.js
+/* eslint-disable */
+import _, { divide } from 'lodash';
+/* eslint-enable */
```

Comment:
### [OPTIONAL]
- [ ] Please ensure that you avoid disabling linters as they are there to help you write better code.
- [ ] Kindly avoid using any external libraries at this stage, because the tasks are there to make sure you master Javascript